lib/convection/model/template/resource_property/aws_lambda_dead_letter_config.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../resource_property'
+
+module Convection
+  module Model
+    class Template
+      class ResourceProperty
+        # Represents an {http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-deadletterconfig.html
+        # Lambda Function DeadLetterConfig}
+        class LambdaDeadLetterConfig < ResourceProperty
```

Comment:
I think the more proper name for this would be LambdaFunctionDeadLetterConfig, and the filename should also reflect that, a la LambdaFunctionCode in the aws_lambda_function_code.rb file.  In contrast, LambdaVpcConfig was done incorrectly based on the standard used in most places of reflecting amazon's naming as closely as possible.